lib/ret/hub.ex

```diff
@@ -87,9 +88,16 @@ defmodule Ret.Hub do
 
     hub
     |> cast(%{max_occupant_count: new_max_occupant_count}, [:max_occupant_count])
+    |> maybe_add_last_active_at_to_changeset(occupant_count)
     |> validate_required([:max_occupant_count])
   end
 
+  # NOTE occupant_count is 1 when there is 1 *other* user in the room with you, so active is when >= 1.
+  defp maybe_add_last_active_at_to_changeset(changeset, occupant_count) when occupant_count >= 1,
```

Comment:
Lol I read the comment, then read the next line and was still confused by the `occupant_count >= 1`